buidlerConfigSol4.js

```diff
@@ -149,7 +149,7 @@ module.exports = {
     tests: "./test/sol4"
   },
 
-  mocha: {
-    enableTimeouts: false
+  mocha: {	
```

Comment:
adding spaces

probably not required

Reply:
this comment is for last commit only.
here can't see the extra spaces.
just check spaces in editor.